aces/Loris
# [Test] unittests for NDB_BVL_Instrument library

Unit tests for the NDB_BVL_Instrument library under `php/libraries`. This PR aims to complete coverage for this library.

#### Testing instructions

1. Run `npm run tests:unit -- --filter NDB_BVL_Instrument_Test` or check out the Travis output
